### ISSUE TITLE
Deleted records not being removed from ES pages

### ISSUE THREAD

**Issue by Mike-Averill** [BiologicalRecordsCentre/iRecord#1543]
Deleted records cannot be removed from the search display even though the details are not behind the record  (see Record 32126862)

**Comment by DavidHepper** [comment]
'Search' is powered by indexing. Following a delete, until the indexer next runs the record stays in the hits unless extra work is done to exclude. If you can send me a URI that still links to 32126862 please let me know.

**Comment by Mike-Averill** [comment]
Was this aimed at me David? If you search https://irecord.org.uk/all-records and use the ID 32126862, the ghost of the record is still there but no detail behind it.


Maybe it doesn't matter if it doesn't get picked up in downloads


cheers

**Comment by DavidHepper** [comment]
The indexer hasn't run properly. I'd not expect the record details and photo to show in the grid. 
I'll leave this open for others.

**Comment by kitenetter** [comment]
Further example:

During testing I added two record to the vascular plant recording form, record IDs 32255524 and 32255526 (both part of the same sample). These are not genuine records, and after testing I deleted them. They are showing as deleted if I try to find them in the warehouse interface. However, they are still displaying on the "My records" and "All records" pages, and on the verification page.

If I try to edit the records from the "My records" page, I arrive at the vascular plant form, but it is displaying various "Unsupported control" messages.

If I try to edit the records from the verification page I get "The record you are trying to load does not exist".

The records are not visible on the old (non-ES) explore page, so I guess this is a problem with the ES version not getting properly updated following the deletes.
